modules/meta.analysis/R/approx.posterior.R

```diff
@@ -43,7 +43,9 @@ approx.posterior <- function(trait.mcmc, priors, trait.data=NULL, outdir=NULL, f
     
     dat    <- trait.mcmc[[trait]]
     vname  <- colnames(dat[[1]])
-    dat    <- as.matrix(dat)[,'beta.o']
+    if('beta.o' %in% vname) {
```

Comment:
I think `trait.mcmc[[trait]]` should be of class `mcmc.list`. I worry that what we had was a kludge that can silently pass errors.

At least, this is a good place to turn this into a useful error message, e.g. if you add 

```r
...
}  else if (! (beta.o %in% vname)) {
  logger.error("trait.mcmc does not have a value for beta.o, only for ",  colnames(dat))
}
```

Also, if `class(mcmc.list[[trait]])` returns `"vector"`, it is probably because the coda library is not loaded.

```r
library(coda)
class(mcmc.list[[trait]])
```